.github/actions/kind/action.yml

```diff
@@ -7,16 +7,16 @@ runs:
     - name: Set up KinD
       uses: container-tools/kind-action@v1
       with:
-        config: .github/kind-config.yaml
+        config: dev/config/kind/config.yaml
         registry: false                  # Registry not needed
-        node_image: kindest/node:v1.22.2 # Newest image
+        node_image: kindest/node:v1.20.0 # Newest image
     - name: Check cluster status
       shell: bash
       run: |
         kubectl get nodes
         kubectl cluster-info
-    - name: Install Ingres
+    - name: Install Ingress
       shell: bash
       run: |
-        kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.4/deploy/static/provider/kind/deploy.yaml
+        kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
```

Comment:
Why this change? I would prefer to keep the version fixed to reduce CI instability

Reply:
> Just one minor comment, I'm not doing a proper review. My only consideration is that it looks like our scripting "toolkit" is increasing fast so we might consider to cover them with some tests and/or move to a different language/framework that is easier to test (i.e. jbang/picocli but there are many of them). @robpblake Wdyt?

I do see your point. Right now I just aggregated what is currently existing in an "orchestration" script. Not much new.
I guess we could later migrate to jbang or other

Reply:
fallback done